src/client/options/mod.rs

```diff
@@ -1172,16 +1183,25 @@ impl ClientOptions {
         Ok(options)
     }
 
+    /// Creates a `ClientOptions` from the given `ConnectionString`.
+    #[cfg(any(feature = "sync", feature = "tokio-sync"))]
+    pub fn parse_connection_string_sync(conn_str: ConnectionString) -> Result<Self> {
```

Comment:
i might've missed a discussion on this on the original PR but does this method need "sync" in the name (and ditto below)? the [sync version](https://docs.rs/mongodb/latest/mongodb/options/struct.ClientOptions.html#method.parse-1) of `ClientOptions::parse` is named the same as the async one because the feature flags for each are mutually exclusive

Reply:
gotcha, thanks for the explanation!